packages/core/utils/src/traverse/traversals/query-sort.ts

```diff
@@ -116,6 +116,7 @@ const sort = traverseFactory()
     transform: cloneDeep,
 
     remove(key, data) {
+      // eslint-disable-next-line no-unused-vars
       const { [key]: ignored, ...rest } = data;
```

Comment:
why do we need the ` // eslint-disable-next-line no-unused-vars`?

Reply:
Because we're not allowed to create an `ignored` if we don't use it, and we're also not allowed to ignore it with an _ prefix. Could probably be rewritten but here I'm just using code that already existed.